static/gccdump-view.js

```diff
@@ -232,16 +228,19 @@ define(function (require) {
             }
             this.updatePass(this.filters, this.selectize, result.gccDumpOutput);
             this.showGccDumpResults(currOutput);
+
+            // enable UI on first successful compilation or after an invalid compiler selection (eg. clang)
+            if (!this.uiIsReady) {
+                this.uiIsReady = true;
+                this.onUiReady();
+            }
         } else {
             this.selectize.clear(true);
             this.state.selectedPass = '';
             this.updatePass(this.filters, this.selectize, false);
-            this.showGccDumpResults('<no output>');
-        }
-        // enable UI on first compilation
-        if (!this.uiIsReady) {
-            this.uiIsReady = true;
-            this.onUiReady();
+            this.showGccDumpResults('<no output, maybe you switched to a non-GCC compiler ?>');
```

Comment:
Last review I promise. Could you please capitalize the first N? :)

Reply:
No prob ! Done !